cypher-shell/src/integration-test/java/org/neo4j/shell/commands/CypherShellVerboseIntegrationTest.java

```diff
@@ -293,4 +296,30 @@ public void shouldNotContainUnnecessaryNewLines() throws CommandException
                             "%n" +
                             "3 rows available after" ) ) );
     }
+
+    @Test
+    public void useWithNewLine() throws CommandException
+    {
+        assumeTrue( majorVersion( shell.getServerVersion() ) >= 4 ); // :use not supported before 4
```

Comment:
This is correct, right? That `:use` is not supported before 4.0?

Reply:
yes